assets/custom.js

```diff
@@ -1,22 +1,45 @@
-//Shows dropdown box for 'Availability' for desktop
-$(".filter__container").click(function(){
-    const filter__activeContainer = $(this).children().last();
-    filter__activeContainer.toggleClass("display-block");
+//Shows Dropdown box for 'Availability' for DESKTOP
+$(".filter__label").click(function(){
+    const filterActiveContainer = $(this).parent().find('.filter__activeContainer');
+    filterActiveContainer.toggleClass("display-block");
 });
 
-//Shows Menu for mobile when 'Filter and Sort' is clicked
+//Shows Menu for 'Filter and Sort' for MOBILE
 $(".filterAndSortBy__label").click(function(){
     $(this).next().find(".activeContainer").toggleClass("display-block");
     $('.filterAndSortBy__backgroundCover').toggleClass("display-block");
 });
 
-// Hides Menu for mobile when 'Back' is clicked
+// Hides Menu when 'Back' is clicked for MOBILE
 $(".activeContainer__backContainer").click(function(){
     $(".activeContainer").toggleClass("display-block");
     $('.filterAndSortBy__backgroundCover').toggleClass("display-block");
 })
 
-// Shows 'availablity' drop down for mobile
+// Hides Menu container when user clicks outside of container for DESKTOP and MOBILE
+$(document).mouseup(function(e){
+    // Desktop Filter
+    const container = $(".filter__activeContainer");
+    if (!container.is(e.target) && container.has(e.target).length === 0) 
+    {
+        container.removeClass("display-block");
+    }
+    // Mobile Menu
+    const mobileContainer = $(".filterAndSortBy__activeContainer");
+    if (!mobileContainer.is(e.target) && mobileContainer.has(e.target).length === 0) 
+    {
+        mobileContainer.removeClass("display-block");
+        $(".filterAndSortBy__backgroundCover").removeClass("display-block");
+    }
+});
+
+// Shows 'Availability' sub-menu for MOBILE and adjusts chevron accordingly
 $(".availability-label").click(function(){
     $(this).next().toggleClass("display-block");
```

Comment:
would prefer to not use `.next()` - here you can use `$(this).parent().find('.filter__listContainer--mobile')`

also, this might be a nitpick but from looking at the html, usually for an Accordion component like this, the section that displays (`.filter__listContainer--mobile`) when clicking on `Availability` (`.activeContainer__menuItem.availability-label`) should be nested inside that div. 

So `.filter__listContainer--mobile` should be nested inside `.activeContainer__menuItem.availability-label`.

Reply:
Understood. Will change that.

Reply:
Actually, I'd rather leave the two divs as siblings than as a descendant. I would have to go in and redo flex into grid rows and columns. I'm using flex space between for the div containing the label and chevrons `(.activeContainer__menuItem)`.

Reply:
that's cool. now that i'm looking at it, i would just wrap both of those divs inside another div, just so they are grouped together (the code just looked weird to me because they weren't grouped together in one `div`). you don't have to actually make the change but something to keep in mind for the future